packages/expo-updates/README.md

```diff
@@ -118,7 +118,7 @@ In `expo-updates@0.9.0` and above, we support automatic installation of the modu
 
 # Removing pre-installed expo-updates
 
-Projects created by `expo init` and `expo prebuild` come with expo-updates pre-installed, because we anticipate most users will want this functionality. However, if you do not intend to use OTA updates, you can disable or uninstall the module.
+Projects created by `expo init` and `npx expo prebuild` come with expo-updates pre-installed, because we anticipate most users will want this functionality. However, if you do not intend to use OTA updates, you can disable or uninstall the module.
```

Comment:
```suggestion
Projects created by `npx create-expo-app` and `npx expo prebuild` come with expo-updates pre-installed, because we anticipate most users will want this functionality. However, if you do not intend to use OTA updates, you can disable or uninstall the module.
```